.github/workflows/dockerhub.yml

```diff
@@ -0,0 +1,25 @@
+name: Push to DockerHub
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  dockerhub:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - uses: satackey/action-docker-layer-caching@v0.0.5
+
+      - name: Build and push Docker images
+        uses: docker/build-push-action@v1.1.0
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
```

Comment:
Were you able to add these secrets yourself, or has @ErikSchierboom added them for you?

Reply:
I believe they're global, organization-wide secrets, right @iHiD?

Reply:
The keys are org-wide and have been added to all three python tooling repos 🙂

Reply:
The first push to Dockerhub needs to be manual though I believe. @ErikSchierboom Is this something you need to do (probably)?

Reply:
No I don't, as that has already been done :)